docs/detections/value-list-exceptions.asciidoc

```diff
@@ -17,11 +17,10 @@ TIP: You can also use a value list as the <<indicator-value-lists,indicator matc
 [float]
 [[manage-value-lists]]
 === Create value lists
-When creating a value list for a rule exception, be mindful of the list's size and data type. In general, all rule types support value list exceptions, but there are some limitations for especially large lists or certain data types. The following value list types can _only_ be used with custom query, machine learning, and indicator match rule types:
+When creating a value list for a rule exception, be mindful of the list's size and data type. In general, all rule types support value list exceptions, but there are some limitations for especially large lists or certain data types. The following limitations only apply to value lists being used for custom query, machine learning, and indicator match rules:
```

Comment:
Thanks for the updates! Would "limitations" be the right word here? We're saying these rule types can handle lists with more than 65, 536 values and IP range lists with more than 200 dash notation values. The limitations are on the rule types not listed.

Reply:
Hm, I can't remember why I worded it this way. Let me know if the following correctly restates what you're saying:

- Value lists for custom query, machine learning, and indicator match rules can have more than 65,536 values if they're of the `Keyword` or the `IP address` type. 
- Value lists for custom query, machine learning, and indicator match rules can have over 200 dash notation values, or more than 65,536 CIDR notation values, if they're of the `IP range` type.

Reply:
Potential draft

```suggestion
When you create a value list for a rule exception, be mindful of the list's size and data type. All rule types support value list exceptions, but extremely large lists or certain data types have limitations. 

Value lists for exceptions that you're adding to custom query, machine learning, and indicator match rules can support the following:
* Lists of the **Keywords** or **IP addresses** types can have more than 65,536 values.
* Lists of the **IP ranges** type can have over 200 dash notation values, or more than 65,536 CIDR notation values.
```